hw1/02-palindrome.js

```diff
@@ -1,3 +1,34 @@
 const elem = document.querySelector('input');
 
 elem.addEventListener('input', handleInput);
+const res = document.getElementById('result');
+
+
+function handleInput(){
+
```

Comment:
not sure why these empty spaces but it'd be great is you stay consistent with spacing!

Reply:
Sure! That's a great observation, will keep it in mind!